.github/workflows/release.yml

```diff
@@ -200,10 +200,12 @@ jobs:
     needs: matrix_prep
     strategy:
       fail-fast: false
-      matrix: ${{ fromJson(needs.matrix_prep.outputs.matrix) }}
-    runs-on: [self-hosted, macos-10.15]
+      matrix:
+       from_matrix_prep: ${{ fromJson(needs.matrix_prep.outputs.matrix) }}
+       runs_on: [[self-hosted, macos-10.15], macos-latest]
```

Comment:
GitHub Action only supports multiple `runs-on` using `strategy`. In order to minimize the changes, I change the original `matrix` to `matrix.from_matrix_prep` and `matrix.runs_on`.

However, I don't know if GitHub Action even recognizes `[[self-hosted, macos-10.15], macos-latest]`.

Reply:
Hi, sorry for the late reply, the notification is somehow lost for no reason 😅

macOS are generally backward compatible so we don't need to build a whole matrix of wheels here (with 10.14 and the M1 chip transition as exceptions, which already addressed by `build_and_test_1014` and `build_and_test_m1` task respectively), but we do need to ensure built wheels are behaving normally on newer systems, so IMHO these should be moved to `testing.yml`